.devcontainer/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM debian:10
+
+# This Dockerfile adds a non-root 'vscode' user with sudo access. However, for Linux,
+# this user's GID/UID must match your local user UID/GID to avoid permission issues
+# with bind mounts. Update USER_UID / USER_GID if yours is not 1000. See
+# https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=azure-sdk-for-cpp
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+ARG PORT=4000
+
+# Install packages as root
+USER root
+
+# Install needed packages and setup non-root user. Use a separate RUN statement to add your own dependencies.
+RUN apt-get update \
```

Comment:
Although its convenient for size of the image (reduced layers) to run everthing as 1 command its hard to debug when it fails.  You might consider breaking this into a couple RUN steps broken up by purposes "Create user" "Install packages" "Setup\Cleanup Environment and temp files"

Reply:
I tried by `RUN` will be executed in parallel to any other `RUN` and we need sequential execution here.